api/config/config_local.json

```diff
@@ -13,7 +13,9 @@
     "vpcServicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
     "timeoutInSeconds": 60,
     "jupyterDockerImage": "broadinstitute/terra-jupyter-aou:0.0.1",
-    "welderDockerImage": "broadinstitute/welder-server:7ae1a62"
+    "welderDockerImage": "broadinstitute/welder-server:7ae1a62",
+    "shibbolethApiBaseUrl": "https:\/\/profile-dot-broad-shibboleth-prod.appspot.com/dev",
```

Comment:
Yes – this is how the new Shibboleth service was designed. We noted this some time ago, and no changes were made.

This design also impacts our (in)ability to test against perf. AFAICT, there is no connection possible to the perf environment. So I just pushed a change which configures these params to empty strings in perf (which IMO is probably best, so any attempt fails hard rather than appears to work but acts on the wrong environment).

There's probably some architectural discussions worth having with the Terra UI team once they fully take on ownership of this service.